setup.cfg

```
@@ -20,25 +20,31 @@ description = pyAFQ: Automated Fiber Quantification ... in Python
 long_description = file:README.md
 long_description_content_type = text/markdown
 platforms = OS Independent
+version = attr: pyAFQ.__version__
```

@johndromero and I found that we cannot install this unless we change here to: 

```suggestion
version = attr: AFQ.__version__
```